presenter/src/main/java/com/mashup/presenter/ui/main/BottomBarScreen.kt

```diff
@@ -0,0 +1,29 @@
+package com.mashup.presenter.ui.main
+
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Home
+import androidx.compose.material.icons.filled.Person
+import androidx.compose.material.icons.filled.Settings
+import androidx.compose.ui.graphics.vector.ImageVector
+
+sealed class BottomBarScreen(
+    val route: String,
+    val title: String,
+    val icon: ImageVector
+){
+    object Home: BottomBarScreen(
+        route = "home",
+        title = "Home",
+        icon = Icons.Default.Home
+    )
+    object Profile: BottomBarScreen(
+        route = "Profile",
```

Comment:
다른애들은 route가 다 소문자인데 얘만 대문자입니다!!
```suggestion
        route = "profile",
```